ksqldb-engine/src/test/java/io/confluent/ksql/internal/StorageUtilizationMetricsReporterTest.java

```diff
@@ -253,6 +267,58 @@ public void shouldIgnoreNonSSTMetrics() {
     // Then:
     assertThrows(AssertionError.class, () -> verifyAndGetRegisteredMetric(TASK_STORAGE_METRIC, TASK_ONE_TAGS));
   }
+  
+  @Test
```

Comment:
can you add a case for max task when there are 0 tasks?

Reply:
yup